src/libraries/System.Text.Json/Common/JsonSourceGenerationOptionsAttribute.cs

```diff
@@ -132,5 +132,11 @@ public JsonSourceGenerationOptionsAttribute(JsonSerializerDefaults defaults)
         /// Specifies the source generation mode for types that don't explicitly set the mode with <see cref="JsonSerializableAttribute.GenerationMode"/>.
         /// </summary>
         public JsonSourceGenerationMode GenerationMode { get; set; }
+
+        /// <summary>
+        /// Instructs the source generator to default to <see cref="JsonStringEnumConverter"/>
+        /// instead of numeric serialization for all enum types encountered in its type graph.
+        /// </summary>
+        public bool UseStringEnumConverter { get; set; }
```

Comment:
Does an equivalent option not make sense on JsonSerializerOptions?  My understanding is we're now trying to keep them in sync for any options that aren't specifically focused on one mode or the other.